strawberryfields/backends/tfbackend/ops.py

```diff
@@ -775,9 +794,18 @@ def cubic_phase(
     return output
 
 
-def beamsplitter(t, r, mode1, mode2, in_modes, D, pure=True, batched=False):
+def beamsplitter(theta, phi, mode1, mode2, in_modes, D, pure=True, batched=False):
+    """returns beamsplitter unitary matrix on specified input modes"""
+    theta = tf.cast(theta, def_type)
+    phi = tf.cast(phi, def_type)
+    matrix = beamsplitter_matrix(theta, phi, D, batched)
+    output = two_mode_gate(matrix, mode1, mode2, in_modes, pure, batched)
+    return output
+
+
+def two_mode_squeezer(r, theta, mode1, mode2, in_modes, D, pure=True, batched=False):
     """returns beamsplitter unitary matrix on specified input modes"""
-    matrix = beamsplitter_matrix(t, r, D, batched)
+    matrix = two_mode_squeezer_matrix(r, theta, D, batched)
     output = two_mode_gate(matrix, mode1, mode2, in_modes, pure, batched)
```

Comment:
This function is also missed by coverage. Would be nice to have a test, but since the scope of the PR merely exchanged a part of the logic in here, it's alright to leave it as is.

Reply:
Hmm.. 🤔 How come it's missed? Wasn't there already a test for the `two_mode_squeezer_matrix`?

Reply:
No, it was missed previously